developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas

```diff
@@ -581,7 +582,10 @@ function TProjectFile.GetOwnerProject: TProject;
     if Assigned(FParent) then
       Result := FParent.OwnerProject;
     if Result = nil then
+    begin
+      // TODO: RAISE ERROR
```

Comment:
This is a lowpri item -- technically this shouldn't happen, but the fallback we have here is probably fine.